server/channels/app/custom_profile_attributes.go

```diff
@@ -269,3 +269,21 @@ func (a *App) PatchCPAValues(userID string, fieldValueMap map[string]json.RawMes
 
 	return updatedValues, nil
 }
+
+func (a *App) DeleteCPAValuesForUser(userID string) *model.AppError {
```

Comment:
Are we going to have DeleteCPAValuesFor<SomethingElse> or should we align with smth. like PatchCPAValue ,that is also "for a user"?

Reply:
Makes sense, will update the function name